packages/experimental/Shimmer/src/Shimmer.tsx

```diff
@@ -84,6 +85,8 @@ export const Shimmer = compose<ShimmerType>({
             );
           } else if (element.type == 'circle') {
             rows.push(<Circle key={i} r={element.height / 2} cx={element.xPos} cy={element.yPos} />);
+          } else {
+            assertNever(element.type);
```

Comment:
Nice to add a use case elsewhere in the repo so we have more code to reference =)

Reply:
Unfortunately this was the only place where it seemed to be applicable, because a lot of other cases are really flexible 😓